docs/src/main/asciidoc/amazon-lambda-http-guide.adoc

```diff
@@ -35,4 +35,110 @@ Quarkus project, you'll need to configure your application for deployment to Ama
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-amazon-lambda-http</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-resteasy</artifactId>
+</dependency>
+----
+
+NOTE: This example assumes the project is configured to use the BOM provided by the Quarkus project.
+
+Assuming you have properly configured your project to use Quarkus, this is almost all that is necessary.  Now dev mode is available
+as one would expect with a Quarkus application but, additionally, you can use
+https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-CLI-command-reference-sam-local-start-api.html[SAM local]
+if you are more comfortable with that approach.
+
+=== Existing Projects
+
+For existing projects, the following dependency will need to be added:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-amazon-lambda-http</artifactId>
+</dependency>
+----
+
+It's likely there is already an HTTP dependency defined to serve your application.  If not, however, the following
+dependency can also be added:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-resteasy</artifactId>
+</dependency>
+----
+
+Assuming the application is already configured with a YAML template for the SAM CLI to use, you should be almost ready
+to begin using Quarkus.  One step remains, however.  Your template file should look something like
+https://github.com/awslabs/serverless-application-model/blob/master/examples/apps/hello-world/template.yaml[this].
+
+=== Updating the SAM template file
+
+In an existing AWS project, there will likely already be a YAML file for the SAM CLI to use when developing locally and
+deploying.  However others will need an initial template to work with.  To create that initial template, the following
+plugin can be run from the root of the project:
+
+[source,shell]
+----
+mvn io.quarkus:quarkus-amazon-lambda-http-maven:configure-aws-lambda
+----
+
+If there is no existing template file one will be created with the default name of `template.yaml` and the resource name
+will default to `Quarkus`.  These names might not be preferable or there might be an existing template file to work with.
+In this case, there are two properties that can configure these values:
+
+|===
+|Name |Description
+
+|amazon-lambda-http.template| The template file to use
+|amazon-lambda-http.resource| The resource to manage
+|===
+
+NOTE: Running this plugin in an existing project will attempt to update the existing configuration.  If the template
+file has a different name, the plugin will assume there is no configuration and will generate a new file.
+
+This plugin will update the `Handler` and `Runtime` values for the named resource.  Running this plugin once might be
+sufficient.  However, you might choose to always run this plugin to make sure your template file is always correct.
```

Comment:
`always` is used twice in the same sentence, maybe it should be written a little differently?

Reply:
done